src/components/Pagination/Pagination.js

```diff
@@ -1,19 +1,72 @@
 import './Pagination.scss'
 import React from 'react';
+import PropTypes from 'prop-types'
+
+Pagination.propTypes = {
+    pagination: PropTypes.object.isRequired,
+    onPageChange: PropTypes.func
+}
+
+Pagination.defaultProps = {
+    onPageChange: null
+}
+
+function Pagination(props) {
+    const { pagination, onPageChange } = props;
+    const { _page , _limit , _totalRows } = pagination;
+    const totalPages = Math.ceil(_totalRows / _limit)
+
+    function handlePageSize() {
+        if (Number.isInteger(totalPages)) {
+            if (Number(totalPages) > 7) {
+                if(_page > 4) {
+                    return [...Array(_page + 4 ).keys()].slice(_page - 3)
+                } else {
+                    return [...Array(8).keys()].slice(1)
+                }
```

Comment:
nếu if mà đã return rồi. Thì k cần else nữa nhé, vì khi return thì hàm đó đã kết thúc luôn rồi, nó k chạy nữa
```suggestion
                if(_page > 4) {
                    return [...Array(_page + 4 ).keys()].slice(_page - 3)
                }
                return [...Array(8).keys()].slice(1)
```
Tương tự với những cái khác